### ISSUE TITLE
logs highlighted in scratch window only after Rider 2017.3 restarts

### ISSUE THREAD

**Issue by karolis-k** [JetBrains/ideolog#36]
1. create 'log' scratch.
2. paste text.
3. observe that it's not highlighted.
4. restart Rider.
5. when same window opens - log text is properly highlighted.

**Comment by smirok** [comment]
Hi! I close the issue because it seems to be no longer relevant.

Please notify me if it's still actual for you.
